website/pages/en/help.js

```diff
@@ -46,7 +46,12 @@ class Help extends React.Component {
                         <header className="postHeader">
                             <h1>Need help?</h1>
                         </header>
-                        <p>This project is maintained by a dedicated group of people.</p>
+                        <p>This project is maintained by a dedicated group of people. Some are available for 1:1 help. Reach out to them:</p>
```

Comment:
Can we change the wording to include free 1:1 coding session through the [Open Office Hours](https://calendly.com/webdriverio/open-office-hours). Just to ensure that there are 2 types of support.

Reply:
Yep! Will do

Reply:
Updated it to look like this:

![image](https://user-images.githubusercontent.com/706039/92277566-d6998580-eeb8-11ea-8123-d42b4ededf74.png)

I think that works better than having it in the leading paragraph, but I'm happy to move it if you'd prefer.

Reply:
Thanks for the update Kevin. I feel like it makes sense to also highlight that the Office Hours are for support only on contributing to WebdriverIO and the paid support is about consulting of any kind. I would like to avoid that people see the office hours as a chance to get help on their company projects.

Reply:
Maybe I can split it up:

> This project is maintained by a dedicated group of people.
> 
> If you're interested in making a contribution to the WebdriverIO project, check out the free "Office Hour" program to help you with your efforts.
> 
> If you're seeking immediate help with your own project, reach out to a WebdriverIO expert:
> 
> - List of names...
> - Add your name
> 
> Otherwise, check out these free resources:
> [Docs/Gitter/Blog columns go here]

Reply:
@klamping sounds good to me 👍

Reply:
Updated:

![image](https://user-images.githubusercontent.com/706039/92523345-cf8ab400-f1e5-11ea-92a1-f32e1440db15.png)